Climate_police/tests/hw7-mao/pollution_map.py

```diff
@@ -0,0 +1,44 @@
+import plotly.offline as py
+py.init_notebook_mode()
+
+# import pandas as pd
+
+from pre_process import pre_process
+
+def pollution_map(df, source, year):
```

Comment:
The next step for our project could be to implement this function in order to make it clickable and let customers choose the data to visualize!!!